### PR TITLE
[Meal] Create meal screen

### LINKED ISSUE #41: [Meal] Create meal screen
## Overview (Required)

- Create meal screen

## Todo

- [x] Create meal screen view
- [x] Create meal business logic

### PR DESCRIPTION
**Pull request by wsi1212**
## Overview (Required)

-  Create meal screen

## Video
https://github.com/Team-B1ND/dodamdodam-android/assets/105199741/c6f57636-921a-4d15-852b-ac3ede3ba5e6

https://github.com/Team-B1ND/dodamdodam-android/assets/105199741/454758cd-e964-4309-8e86-bae3bbfe57c5

## Issue

- close #41

### CODE REVIEW

**Comment by wsi1212** [comment]
LGBT 🏳️‍🌈

File: feature/meal/src/main/java/com/b1nd/dodam/meal/MealScreen.kt [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+//                            style = MaterialTheme.typography.bodyMedium,
+//                            color = MaterialTheme.colorScheme.tertiary,
+//                        )
+                    }
+                }
+            }
+        } else {
+            Column {
+                val widths = listOf(150, 100, 75, 120, 60, 80)
+                repeat(6) { index ->
```
Comment (mercen-lee) [inline feature/meal/src/main/java/com/b1nd/dodam/meal/MealScreen.kt]: why not ```widths.forEach``` ??????

**Review by mercen-lee — CHANGES_REQUESTED** [review]

File: feature/meal/src/main/java/com/b1nd/dodam/meal/MealScreen.kt [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+                        Text(
+                            modifier = Modifier.align(Alignment.CenterStart),
+                            text = "급식",
+                            style = MaterialTheme.typography.titleLarge,
+                            color = MaterialTheme.colorScheme.onBackground
+                        )
+                    }
+                }
+                if (!uiState.isLoading) {
+                    items(uiState.meal.size) { index ->
```
Comment (mercen-lee) [inline feature/meal/src/main/java/com/b1nd/dodam/meal/MealScreen.kt:132]: Please remove duplicate code with this variable
```kotlin
val isToday = index == 0
```

**Review by mercen-lee — CHANGES_REQUESTED** [review]

**Review by mercen-lee — APPROVED** [review]
LGTM 👍

**Review by SeokgyuYun — APPROVED** [review]
LGTM 😀

**Pull request merged**